kb/plane_geometry/formal_statements/statements/statements_scs/stat_straight_line_parallelism_intersection_segment.scs

```diff
@@ -0,0 +1,14 @@
+stat_straight_line_parallelism_intersection_segment <- statement;;
+
+stat_straight_line_parallelism_intersection_segment -> rrel_key_sc_element: concept_straight_line; nrel_parallelism; nrel_intersection; concept_segment;;
+
+stat_straight_line_parallelism_intersection_segment => nrel_main_idtf: [Акс.(прямая;параллелизм*;пересечение*;отрезок)] (*<- lang_ru;;*);;
+stat_straight_line_parallelism_intersection_segment => nrel_main_idtf: [Ax.(straight line;parallelism*;intersection*;segment)] (*<- lang_en;;*);;

```

Comment:
It isn't an axiom. It is a statement. You should use [Stat.(...)] insted of [Ax.(...)]